src/python/dxpy/templating/templates/nextflow/dxapp.json

```diff
@@ -42,6 +42,27 @@
       "help": "Shows additional information in the job logs. If true, the execution log messages from Nextflow will also be included.",
       "class": "boolean",
       "default": false
+    },
+    {
+      "name": "resume",
+      "label": "Resume",
+      "help": "Enables resume functionality in Nextflow workflows.",
+      "class": "boolean",
+      "default": "false"
+    },
+    {
+      "name": "resume_session",
+      "label": "Resume Session",
+      "help": "Session to be resumed",
+      "class": "string",
+      "optional": "true"
+    },
+    {
+      "name": "no_future_resume",
```

Comment:
I'm wondering why the input is called "no_future_resume"? It suggests that we don't want to allow for future resumes to happen from the current run (but the help suggests otherwise). Do I understand right?

Reply:
It is copy pasted from https://confluence.internal.dnanexus.com/pages/viewpage.action?spaceKey=PRODUCT&title=Nextflow+v1

but you are right the help message is somehow confusing...Will update it. Thanks!

Reply:
This is much clearer, thanks!